commands/changed/README.md

```diff
@@ -27,3 +27,12 @@ package-2
 Unlike `lerna ls`, however, `lerna changed` **does not** support [filter options](https://www.npmjs.com/package/@lerna/filter-options), as filtering is not supported by `lerna version` or `lerna publish`.
 
 `lerna changed` also supports all the flags supported by [`lerna version`](https://github.com/lerna/lerna/tree/master/commands/version#options), but the only relevant one is [`--ignore-changes`](https://github.com/lerna/lerna/tree/master/commands/version#--ignore-changes).
+
+### `--include-merged-tags`
```

Comment:
This duplicated options block is not necessary. Adding a link to the docs under `commands/version` directly is fine.